protocol/protobuf/pairing.proto

```diff
@@ -255,3 +267,16 @@ message SyncContactRequestDecision {
     DECLINED = 1;
   }
 }
+
+/* `SyncProfile` is used to describe profile of logged in user */
+message SyncProfile {
+  string key_uid = 1;
+  string display_name = 2;
```

Comment:
Display name I believe could be added to `SyncSetting`, for consistency, instead of creating a new protobuf.
Images are a bit trickier, but I defer to @Samyoul

Reply:
Idea was to create a new buffer which refers to profile, to combine display name and identity images.